src/user/messages.js

```diff
@@ -39,6 +39,17 @@ class UserMessages {
         }
       }
 
+    _.showListDesa
+      = (data) =>
+      {
+        var ret = 'Desa Terdekat\n\n';
+        const listDesa = data.results.bindings;
+        listDesa.forEach(function (desa) {
+          ret += desa.kodeDesa.value + ' - ' + desa.desaLabel.value + '\n';
+        });
+        return ret;
```

Comment:
Did you know that the application use non-strict NodeJS? I mean, it's no need to add semi-colon. Since I didn't use one, it's will be an inconsistency.